### ISSUE TITLE
Shaku

### ISSUE THREAD

**Issue by Muppet1** [demilich1/metastone#352]
Seems to not draw a card if you attack the hero.  Will confirm when I next see it.

**Comment by Muppet1** [comment]
yes its works when you hit a minion and not when you hit the hero

**Comment by doctorpangloss** [comment]
So are you saying Shaku is, or is not broken? Please close the ticket if it's not broken :)

**Comment by Muppet1** [comment]
it is broken it needs to draw a card when you attack the hero too.  I think I fixed mine just by removing the hero part.
